.github/workflows/add-catalog.yml

```diff
@@ -15,19 +17,29 @@ jobs:
         run : |
           echo "MESHERY_CLOUD_BASE_URL=https://meshery.layer5.io" >> $GITHUB_ENV
           echo "MESHERY_CATALOG_FILES_DIR=catalog" >> $GITHUB_ENV
+      - name: Cleanup
+        run: |
+          rm $MESHERY_CATALOG_FILES_DIR/*
+          rm ./collections/_catalog/deployment/*
+          rm ./collections/_catalog/jwt/*
+          rm ./collections/_catalog/observability/*
+          rm ./collections/_catalog/resiliency/*
+          rm ./collections/_catalog/scaling/*
+          rm ./collections/_catalog/security/*
+          rm ./collections/_catalog/traffic-management/*
+          rm ./collections/_catalog/troubleshooting/*
+          rm ./collections/_catalog/wasm/*
+          rm ./collections/_catalog/workloads/*
```

Comment:
@leecalcote 
If this looks good then we can merge and test this out.

The only caveat to deleting all the existing files is that we would also be deleting our old Catalog content which was manually added and is not available in remote provider DB. So we won't necessarily get that. But at least this would make it consistent with https://meshery.layer5.io/catalog